src/handlers/motions/motionFinalized/helpers.ts

```diff
@@ -375,3 +390,111 @@ export const updateColonyUnclaimedStakes = async (
     });
   }
 };
+
+export const claimExpenditurePayouts = async (
+  action: string,
+  colonyAddress: string,
+): Promise<void> => {
+  const colonyClient = await getCachedColonyClient(colonyAddress);
+
+  const parsedAction = parseAction(action, { colonyClient }) as
+    | TransactionDescription
+    | undefined;
+
+  if (
+    !colonyClient ||
+    !parsedAction ||
+    parsedAction.name !== ColonyOperations.Multicall
+  ) {
+    return;
+  }
+
+  const firstAction = parsedAction.args[0][0];
+
+  let decodedSetExpenditureStateArgs;
+
+  try {
+    decodedSetExpenditureStateArgs = colonyClient.interface.decodeFunctionData(
+      'setExpenditureState',
+      firstAction,
+    );
+  } catch (error) {
+    return;
+  }
+
+  const [, , expenditureId, storageSlot, , keys, value] =
+    decodedSetExpenditureStateArgs;
+  const [slotId] = keys;
+
+  const convertedSlotId = toNumber(slotId);
+  const convertedExpenditureId = toNumber(expenditureId);
+  const convertedValue = toNumber(value);
+  const convertedStorageSlot = toNumber(storageSlot);
+
+  if (!colonyAddress) {
+    output('Colony address missing for ExpenditureStateChanged event');
+    return;
+  }
+
+  // @NOTE: If:
+  // - The value is not 0
+  // - The storage slot is not 26
+  // Then we can assume this state change ain't a stage release
+  if (convertedStorageSlot !== 26 || convertedValue !== 0) {
```

Comment:
I'd maybe add these as `enums`? But maybe that's just me, as I get alarm bells when I see "magic" values  randomly through the code

Reply:
Actually this will be replaced by https://github.com/JoinColony/colonyCDapp/issues/2014, but it's a fair point so I added a const with a comment for now.